components/TopBarProfileMenu.js

```diff
@@ -115,63 +186,113 @@ class TopBarProfileMenu extends React.Component {
-              <FormattedMessage id="menu.myAccount" defaultMessage="My account" />
-            </P>
-            <Box as="ul" p={0} my={2}>
-              <UserMenuLinkEntry href={`/${LoggedInUser.username}`}>
-                <FormattedMessage id="menu.profile" defaultMessage="Profile" />
-              </UserMenuLinkEntry>
-              <Query
-                query={memberInvitationsCountQuery}
-                variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
-                fetchPolicy="network-only"
-              >
-                {({ data, loading }) =>
-                  loading === false && data && data.memberInvitations && data.memberInvitations.length > 0 ? (
-                    <UserMenuLinkEntry href="/member-invitations">
-                      <FormattedMessage
-                        id="menu.pendingInvitations"
-                        defaultMessage="Pending Invitations ({numberOfInvitations})"
-                        values={{ numberOfInvitations: data.memberInvitations.length }}
-                      />
-                    </UserMenuLinkEntry>
-                  ) : null
-                }
-              </Query>
-              <UserMenuLinkEntry href={`/${LoggedInUser.collective.slug}/edit`}>
-                <FormattedMessage id="Settings" defaultMessage="Settings" />
-              </UserMenuLinkEntry>
-              <UserMenuLinkEntry href={`/${LoggedInUser.username}/recurring-contributions`}>
-                <FormattedMessage id="menu.subscriptions" defaultMessage="Manage Contributions" />
-              </UserMenuLinkEntry>
-              <UserMenuLinkEntry href={`/${LoggedInUser.username}/transactions`}>
-                <FormattedMessage id="menu.transactions" defaultMessage="Transactions" />
-              </UserMenuLinkEntry>
-              <UserMenuLinkEntry href="/applications">
-                <FormattedMessage id="menu.applications" defaultMessage="Applications" />
-              </UserMenuLinkEntry>
-              <UserMenuLinkEntry as="a" href="https://docs.opencollective.com">
-                <FormattedMessage id="menu.help" defaultMessage="Help" />
-              </UserMenuLinkEntry>
-              <UserMenuLinkEntry as="a" data-cy="logout" onClick={this.logout}>
-                <FormattedMessage id="menu.logout" defaultMessage="Log out" /> →
-              </UserMenuLinkEntry>
+          <Hide lg md sm>
+            <Box p={3} pb={1} onClick={this.showAccountInfo}>
+              <Flex alignItems="center">
+                {showMyAccount ? (
+                  <P
+                    color="black.700"
+                    fontSize="12px"
+                    fontWeight="500"
+                    letterSpacing="0.06em"
+                    pr={2}
+                    textTransform="uppercase"
+                    whiteSpace="nowrap"
+                    onClick={this.hideAccountInfo}
+                  >
+                    ← <FormattedMessage id="go-back" defaultMessage="Back" />
```

Comment:
Just to be consistent,

```suggestion
                    ← <FormattedMessage id="menu.goBack" defaultMessage="Back" />
```

Reply:
I would re-use the existing one here

```suggestion
                    ← <FormattedMessage id="Back" defaultMessage="Back" />
```